doc/README.md

```diff
@@ -63,7 +61,10 @@ The Dash Core repo's [root README](/README.md) contains relevant information on
 
 ### Resources
 * Discuss on the [Dash Forum](https://dash.org/forum), in the Development & Technical Discussion board.
-* Discuss on [Dash Nation Discord](http://dashchat.org)
+* See the [Dash Developer Documentation](https://dashcore.readme.io/)
```

Comment:
Maybe make "See ..." the first on in the list?

Reply:
done